src/res.rc

```diff
@@ -11,6 +11,7 @@
 
 #define APSTUDIO_HIDDEN_SYMBOLS
 #include <windows.h>
+#include <winnt.h>

```

Comment:
Windows.h includes windef.h which includes winnt.h.  Is this not true for mingw?

Reply:
mingw resource compiler seems not look winnt.h so SUBLANG_XXX is not defined. I found workaround for this now.